posts/2018-05-prometheus.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Prometheus: Pulling Operational Insight Forward
+author: David Antaramian
+status: draft-for-review
+---
+Operational insight is critical to the success of any software deployment, and
+Timber is no exception. We need to know that service is being delivered on a
+consistent basis to our users and that any anomaly is reported to our team
+quickly. There are a multitude of solutions that exist for gaining this type of
+insight, and each has their own benefits and trade-offs as well as cost
```

Comment:
Can probably leave out "cost considerations" here.

Reply:
I'm leaving this in. I view cost considerations as separate to the benefits and trade-offs of the individual feature set. For example, InfluxDB stores individual events on disk, allowing for higher resolution queries. That feature is beneficial, but the cost of storage grows proportionally with the data set. So while it's a beneficial feature, it may be cost-prohibitive to maintain the necessary disk sizes when dealing with extremely large datasets.